src/lib/datatransfers/datatransfer_base.py

```diff
@@ -17,15 +24,115 @@ class TransferJob(CamelModel):
     logs: TransferJobLogs
 
 
+class DataTransferType(str, Enum):
+    """Types of data transfer services"""
+
+    s3 = "s3"
+    wormhole = "wormhole"
+
+
+class DataTransferDirective(CamelModel):
+    transfer_method: Literal[
+        DataTransferType.s3,
+        DataTransferType.wormhole,
+    ]
+
+
+class WormholeDataTransferDirective(DataTransferDirective):
```

Comment:
IIUC, the response to the user is the magicwormhole code (`wormhole_code`), and then the user should know how to use it for data transfer, employing the Python library or the API, correct?

I'm asking because we don't want the user to depend on a programming language to perform the transfer on their side.

If the transfer can be done using the API, then I suggest including the call using `curl` as a helper